### ISSUE TITLE
Permit seeding of ZIM files downloaded via Bittorrent

### ISSUE THREAD

**Issue by TheMageKing** [kiwix/kiwix-lib#424]
This is fairly self-explanatory.  I think that users should be able to set up their Kiwix program to seed files downloaded via BitTorrent.  This would benefit all users significantly, by increasing the number of seeds.  Aria2c includes this functionality: we just need to use it.

**Comment by kelson42** [comment]
@TheMageKing Yes. The last time I tried this, the problem was that the download was not starting quickly even with a webseed in the torrent, but this is years ago. Maybe it would work better now.

**Comment by mgautierfr** [comment]
Mostly agree. But we have to be careful that this is opt-in and there is no issue on this.

Some users may use kiwix in a context where it is somehow dangerous to use it and bypass some local restrictions. We must care that no internet connection is made behind the back of our user (especially seeding content)
